backend/src/main/java/rocks/inspectit/gepard/agentmanager/connection/model/Connection.java

```diff
@@ -17,12 +16,12 @@
 @ToString
 public class Connection {
 
-  /** The id of the connection. */
-  private UUID id;
```

Comment:
Feels weird, that the root of our entity has no ID, but I understand why you´ve removed it.

Reply:
It owns an ID indirectly through the cache key. 
If we would use a repository class to store connections, we would include the id back to the entity.